script/setup/install-dev-tools

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env bash
 
+GOLANGCI_LINT_VERSION="v1.27.0"
+
 #
 # Install developer tools to $GOBIN (or $GOPATH/bin if unset)
 #
 set -eu -o pipefail
 
+# Enable Go modules
+export GO111MODULE=on
```

Comment:
Perhaps inline this for the lines that need this instead of exporting if we only want it for golangci-lint?